### PR TITLE
feat(cli): symbiotic integration

### PR DESCRIPTION
**Pull request by mempirate**

### CODE REVIEW

File: bolt-cli/src/commands/operators.rs [hunk 1]
```diff
@@ -140,22 +186,124 @@ impl OperatorsCommand {
...
+                    }
+
+                    Ok(())
                 }
             },
         }
     }
 }
 
+fn request_confirmation() -> bool {
```
Comment (merklefruit) [inline bolt-cli/src/commands/operators.rs]: we can use inquire for this kind of thing. https://docs.rs/inquire/latest/inquire/struct.Confirm.html

Comment (thedevbirb) [reply bolt-cli/src/commands/operators.rs]: It's cool but considering current usage I'm okay with having a little function and not having another dependency

**Review by thedevbirb — APPROVED** [review]

File: bolt-cli/src/commands/operators.rs (hunk 1 above)
Comment (merklefruit) [reply bolt-cli/src/commands/operators.rs]: Sounds good, however I'm OK adding this dependency down the line. It also enables interactivity in the CLI.

Comment (thedevbirb) [reply bolt-cli/src/commands/operators.rs]: Tracked in an issue!

**Pull request merged**